test.rb

```diff
@@ -0,0 +1,5 @@
+if foo
```

Comment:
__[development][rubocop]__
* Missing frozen string literal comment.
* Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. [:ref](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)
* Use safe navigation (`&.`) instead of checking if an object exists before calling the method.

<!-- Comment by [SideCI](https://sideci.com) -->